### ISSUE TITLE
customize add_row font color

### ISSUE THREAD

**Issue by HassenPy** [JelteF/PyLaTeX#186]
Hi, i noticed that there's no way to customize a row to use a custom font color, was that left on purpose? if not, assign me this issue, i ll work on it.

**Comment by JelteF** [comment]
Nope, no reason other than that the person writing the code didn't need it.
PRs are always welcome.
